acracensor/handlers/querycapture_handler.go

```diff
@@ -192,7 +196,13 @@ func (handler *QueryCaptureHandler) GetForbiddenQueries() []string{
 	}
 	return forbiddenQueries
 }
+func (handler *QueryCaptureHandler) SetSerializationTimeout(timeoutInSeconds time.Duration){
+	handler.serializationTimeoutSeconds = timeoutInSeconds
```

Comment:
Better to expect time.Duration without assumptions of type of value like second/minute/hour. We expect duration and use it as is. If user will want to use seconds, he will pass `time.Second * 2`, or `time.Millisecond * 50`. And we will pass it as is to `time.Sleep`. 
For me timeoutInSeconds tells that I should pass value correct "timeout" as seconds in golang - `time.Second * myValue`. But if we will look above, we expect value without converting and convert it itself by `time.Sleep(handler.serializationTimeoutSeconds * time.Second)`